src/icon-manifest/manifestMerger.ts

```diff
@@ -87,7 +87,7 @@ export function toggleAngularPreset(
   disable: boolean,
   files: models.IFileCollection): models.IFileCollection {
   const icons = files.supported
-    .filter(x => x.icon.startsWith('ng_') && (x.icon.endsWith('_js') || x.icon.endsWith('_ts')))
+    .filter(x => x.icon.startsWith('ng_') && !x.icon.endsWith('2'))
```

Comment:
As long as you concede me the icons... I'll just  updated this to support any number at the end of the string.  😜